include/sysapp/launch.h

```diff
@@ -4,32 +4,87 @@
 /**
  * \defgroup sysapp_launch SYSAPP Launch
  * \ingroup sysapp
+ *
+ * Functions to launch and restart titles or system applications.
+ * SYSAPP Launch allows applications to launch other titles, restart their own
+ * title, and to launch certain system titles.
  * @{
  */
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
+/**
+ * Restarts the current title with new arguments once the running application
+ * quits. Instead of returning to the HOME menu upon exit, the system will start
+ * the same title again.
+ *
+ * \param argc
+ * The number of strings in the pa_Argv array. Passed in to the title's main
+ * function as argc.
+ *
+ * \param pa_Argv
+ * An array of strings to use as arguments. Passed into the title's main
+ * function as argv.
+ *
+ * <!--
+ *    meta: why pa_?
+ *    do we need to include the rpx name in argv[0]?
+ *    behaviour where pa_Argv = NULL?
+ * -->
+ */
 void
 SYSRelaunchTitle(uint32_t argc,
                  char *pa_Argv[]);
```

Comment:
Probably means Physical Address.

Reply:
We'll probably have to try it on-console to verify - my attempt at REing didn't turn up a decent answer. Interestingly, [Loadiine doesn't use the pa_ prefix](https://github.com/dimok789/loadiine/blob/master/src/menu/menu.c#L53). There doesn't seem to be much code online that calls relaunch with anything more than `(0, NULL)` so it's hard to tell what kind of address is actually needed. I'd also be curious to see where the argv can be - is it permissible in normal memory (therefore it gets copied out before the relaunch) or does it need to be somewhere else (`OSAllocFromSystem`?)